ripple/src/main/test/posts.test.js

```diff
@@ -1,7 +1,75 @@
 /* Contains unit tests for posts.js file */
 
 const postsModule = require('../webapp/posts');
+const $ = require('jquery');
+require('../__mocks__/localstorage');
+const sinon = require('../node_modules/sinon/pkg/sinon.js');
 
-test('subtract 1 & 2 to equal -1', () => {
-  expect(postsModule.subtract(1, 2)).toBe(-1);
+test('saveCaption: test #1 Save user input for caption area', () => {
+  localStorage.clear();
+  // Set up our document body
+  document.body.innerHTML =
+    '<div>' +
+    '  <textarea id="modal-caption">This is my caption!!!</textarea>' +
+    '</div>';
+  postsModule.saveCaption();
+  expect(localStorage.getItem("caption")).toBe("This is my caption!!!");
+});
+
+/* Test that checks if localStorage correctly stores values if a user registers as community member. */
+test('createPostElement: test #1 create a post element', () => {
+  localStorage.clear();
+  // Set up our document body
+  const testPostElement = document.createElement('div');
+  testPostElement.classList = "post";
+  var content = `
```

Comment:
Add comment describing what this html is from/what it mocks

Reply:
Fixed